### PR TITLE
Maintenance and updates

### PR DESCRIPTION
**Pull request by TacoTheDank**
also fixed some english

### CODE REVIEW

**Comment by ScreamingHawk** [comment]
Wow that's a lot of commits. 
I like to keep a clean history so would be ideal if you could clean this up a bit. Makes it hard to follow changes otherwise. 
Have a look at [this interactive rebase tutorial](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history). You can squash all the `Update build.gradle` commits together and drop the `Add files via upload` and `Delete gradle-wrapper.jar` which just revert each other. 

If you could send a merge request that only includes dc4694c8e79e5ec12601dbf9f1b5802367e0e6ca I'll take it straight away. For the others I'll need to build and test, which I don't have time for at the moment.

**Comment by TacoTheDank** [comment]
yeah... i did this through github so its not very clean

**Comment by ScreamingHawk** [comment]
Going to close this one as it's now outdated.

**Comment by TacoTheDank** [comment]
hmm yeah ill do it cleaner
